src/actorCounter.test.js

```diff
@@ -0,0 +1,13 @@
+const jsdom = require('jsdom');
+const actorCounter = require('./actorCounter.js');
```

Comment:
[OPTIONAL]

- I would recommend using `inport` and `export` instead of `require` to use any library or from modules.